lib/model/QubitInformationObject.php

```diff
@@ -2462,7 +2462,9 @@ public function getTreeViewSiblings(array $options = [], &$siblingsRemaining = n
 
     public static function addTreeViewSortCriteria($criteria)
     {
-        switch (sfConfig::get('app_sort_treeview_informationobject')) {
+        $setting = QubitSetting::getByName('sort_treeview_informationobject');
+
+        switch ($setting->getValue(['sourceCulture' => true])) {
```

Comment:
I think we could use `sfConfig::get('app_sort_treeview_informationobject__source')` to avoid hitting the DB. See:

https://github.com/artefactual/atom/blob/qa/2.x/lib/model/QubitSetting.php#L102